PDM_Node.X/PDM.c

```diff
@@ -61,12 +62,17 @@ void main(void) {
   init_adc(NULL); // Initialize ADC module
   init_termination(TERMINATING); // Initialize programmable CAN termination
   init_can(); // Initialize CAN
-  init_ad7490(); // Initialize AD7490 external ADC chip
   init_rheo(); // Initialize SPI interface for digital rheostats
 
   //TODO: USB
   //TODO: NVM
 
+  // init ad7490 CS
+  CS_AD7490_LAT = 1;
+  CS_AD7490_TRIS = OUTPUT;
+	init_spi5(1, 16);
+  init_ad7490(ad7490_send_spi); // Initialize AD7490 external ADC chip
```

Comment:
The other thing that my code currently allows is a complete decoupling of transfer protocol and chip initialization.  Theoretically you could switch to a model of the IC with 12C, change the init function you pass in and everything should still work fine. IDK how useful that is but I just thought it was neat lol.